theme_avantgarde/views/customizations.xml

```diff
@@ -653,4 +694,140 @@
+        <attribute name="class" remove="pt48 pb48" separator=" "/>
+        <attribute name="data-masonry-template">alternate_text_image_text</attribute>
+    </xpath>
+    <xpath expr="//section/div" position="replace">
+        <t t-call="website.s_masonry_block_alternation_text_image_text_template"/>
+    </xpath>
+</template>
+
+<template id="s_masonry_block_alternation_text_image_text_template" inherit_id="website.s_masonry_block_alternation_text_image_text_template">
+    <!-- Block #1 -->
+    <!-- Main div -->
+    <xpath expr="(//div[hasclass('o_grid_item')])[1]" position="attributes">
+        <attribute name="style" remove="background-image: linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-2) 100%);" add="--grid-item-padding-x: 40px;" separator=" "/>
+    </xpath>
+    <!-- Title -->
+    <xpath expr="(//div[hasclass('o_grid_item')])[1]/h3" position="replace" mode="inner">
+        ✽<br/>Coming up in March<br/>✽
+    </xpath>
+    <!-- Paragraph -->
+    <xpath expr="(//div[hasclass('o_grid_item')])[1]/p" position="replace"/>
+
+    <!-- Block #2 -->
+    <!-- Main div -->
+    <xpath expr="(//div[hasclass('o_grid_item')])[3]" position="attributes">
+        <attribute name="class" remove="o_cc2" add="o_cc4" separator=" "/>
+        <attribute name="style" remove="background-image: linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-1) 100%);" add="--grid-item-padding-x: 40px;" separator=" "/>
+    </xpath>
+    <!-- Title -->
+    <xpath expr="(//div[hasclass('o_grid_item')])[3]/h3" position="replace" mode="inner">
+        ✽<br/>Our very new collection<br/>✽
+    </xpath>
+    <!-- Paragraph -->
+    <xpath expr="(//div[hasclass('o_grid_item')])[3]/p" position="replace"/>
+</template>
+
+<!-- ======== TIMELINE ======== -->
+<template id="s_timeline" inherit_id="website.s_timeline">
+    <!-- Title -->
+    <xpath expr="//h2" position="replace" mode="inner">
+        Our history
+    </xpath>
+    <!-- Lead -->
+    <xpath expr="//p[hasclass('lead')]" position="replace" mode="inner">
+        Discover the journey that made us the most popular gallery in town.
+    </xpath>
+    <!-- Event 1 -->
+    <xpath expr="(//div[hasclass('card')])[1]//small" position="replace" mode="inner">
+        13/06/2024
+    </xpath>
+    <xpath expr="(//div[hasclass('card')])[1]//h3" position="replace" mode="inner">
+        First Culture Award ®
+    </xpath>
+    <!-- Event 2 -->
+    <xpath expr="(//div[hasclass('card')])[2]//small" position="replace" mode="inner">
+        25/06/2019
+    </xpath>
+    <xpath expr="(//div[hasclass('card')])[2]//h3" position="replace" mode="inner">
+        Arrival of Aline Turner at the head of the Gallery
+    </xpath>
+    <xpath expr="(//div[hasclass('card')])[2]//p" position="after">
+        <img alt="" src="/web/image/website.s_company_team_image_3" class="img img-fluid rounded"/>
+    </xpath>
+    <!-- Event 3 -->
+    <xpath expr="(//div[hasclass('card')])[3]" position="replace"/>
```

Comment:
Because of the example with events on both sides, and the incoming fix of the `s_timeline` layout (https://github.com/odoo/odoo/pull/181050), maybe the second card should be removed instead - and the `w-0` and `w-100` of the last entry should be swapped.

... or keep two simultaneous events here.

Reply:
Thanks, added another event to keep the default layout.